test/samples/solidity/latest_08.sol

```diff
@@ -413,3 +413,25 @@ contract Features_0822 {
         emit Y(5);
     }
 }
+
```

Comment:
Can we separate this as a separate sample with proper args? Everything else looks great

Reply:
Addressed in 0d19096